packages/theme-docs/src/store/index.js

```diff
@@ -15,6 +15,29 @@ export const getters = {
   settings (state) {
     return state.settings
   },
+  githubUrls (state) {
+    const { github = '', githubApi = '' } = state.settings
+
+    // GitHub Enterprise
+    if (github.startsWith('http') && githubApi.startsWith('http')) {
```

Comment:
Is it enough to check just existence of the 2 settings?

```javascript
 if (github && githubApi) { ... }
```

Reply:
It should be enough I guess